src/Front/Scanner.x

```diff
@@ -348,6 +366,16 @@ gotoStartLine _ _ = do
   alexSetStartCode lineStart
   alexMonadScan
 
+extractMargin :: AlexUserState -> Int
```

Comment:
Can you document these functions? Use the `-- | documentation` Haddock comments as the other functions